src/app/components/CreateConnectorWizard/CreateConnectorWizardContext.tsx

```diff
@@ -153,7 +166,9 @@ export const useClustersMachine = () => {
     clusterRef,
     useCallback(
       (state: EmittedFrom<typeof clusterRef>) => ({
-        selectedId: state.context.selectedCluster?.id,
+        selectedId: state.context.duplicateMode
```

Comment:
is ternary operator needed here?

Reply:
I think yes, `(property) selectedInstance?: KafkaRequest | undefined`